src/formatter.fs

```diff
@@ -4,6 +4,10 @@ open System
 open System.IO
 open Options.Globals
 
+let private formatPrefix forceUppercase = function
```

Comment:
Can we remove this force uppercase argument?
I'm not certain it makes sense to have the uppercase only in one case.

The callers could call touppercase on the output of the function.